ebill_postfinance/models/account_move.py

```diff
@@ -0,0 +1,139 @@
+# Copyright 2019 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import base64
+import logging
+
+import odoo
+from odoo import _, api, fields, models
```

Comment:
```suggestion
from odoo import api, fields, models
```

`self.env._` everywhere instead